recipes-browser/cog/cog.inc

```diff
@@ -54,10 +54,15 @@ INSANE_SKIP_${PN} = "dev-so"
 COG_DBUS_OWN_USER ?= ""
 
 # Use WebKitGTK+ instead of WPEWebKit
+# TODO: (deprecated) Last cog version using this flag is 0.8.X 
 PACKAGECONFIG[webkitgtk] = "-DCOG_USE_WEBKITGTK=ON,-DCOG_USE_WEBKITGTK=OFF"
 # Expose remote control interface on system bus.
 PACKAGECONFIG[dbus] = "-DCOG_DBUS_SYSTEM_BUS=ON -DCOG_DBUS_OWN_USER=${COG_DBUS_OWN_USER},-DCOG_DBUS_SYSTEM_BUS=OFF"
 # Use wpebackend-fdo.
+# TODO: (deprecated) Last cog version using this flag is 0.10.X 
 PACKAGECONFIG[fdo] = "-DCOG_PLATFORM_FDO=ON,-DCOG_PLATFORM_FDO=OFF,wpebackend-fdo"
 PACKAGECONFIG[drm] = "-DCOG_PLATFORM_DRM=ON,-DCOG_PLATFORM_DRM=OFF,wpebackend-fdo libdrm virtual/libgbm libinput"
+PACKAGECONFIG[wl] = "-DCOG_PLATFORM_WL=ON,-DCOG_PLATFORM_WL=OFF,wpebackend-fdo"
```

Comment:
Should this move to cog_0.11 or some such?

Reply:
You are right. I've moved this to the _devupstream overrides in the 0.10 recipe.